JsonLogic/Rules/ReduceRule.cs

```diff
@@ -70,7 +76,9 @@ protected internal ReduceRule(Rule input, Rule rule, Rule initial)
 				Current = element,
 				Accumulator = accumulator
 			};
+#pragma warning disable IL2026, IL3050
 			var item = JsonSerializer.SerializeToNode(intermediary, _options);
```

Comment:
Would a '_options.WriteToNode' helper be useful for type safety in cases like this?

Reply:
Yes.  Yes it would.